pallets/circuit/Cargo.toml

```diff
@@ -37,7 +37,7 @@ sp-core = { git = "https://github.com/paritytech/substrate", branch = 'polkadot-
 orml-tokens = { git = "https://github.com/t3rn/open-runtime-module-library", branch = "polkadot-v0.9.17", default-features = false }
 orml-traits = { git = "https://github.com/t3rn/open-runtime-module-library", branch = "polkadot-v0.9.17", default-features = false }
 
-[dev-dependencies]
+#[dev-dependencies]
```

Comment:
To fix this:

- remove `default-features = false,` from `dev-dependencies`
- dont add deps to features = ['std']
- uncomment [dev-dependencies]

if it fails to compile because some dep is missing, promote to normal dependencies

Reply:
Fixed on the latest commit